backend_tsc/src/app.ts

```diff
@@ -0,0 +1,74 @@
+import express, {Express, Request, Response} from "express";
+import dotenv from "dotenv";
+import catalogue_router from "./routes/catalogue";
+import {login_router, access_control_router} from "./routes/access_control";
+import sanitize from "express-mongo-sanitize"
+import site_router from "./routes/site";
+import period_router from "./routes/period";
+import region_router from "./routes/region";
```

Comment:
I think these routers could be chucked into a single file and exported from there. idk @Martyrshot @DanielleDPowell  - any input on this? I think it just makes for cleaner looking code.

Reply:
I tried this out (because it was easy), And given that we may need specialty routes as time goes on, putting them al in one file would suck to maintain. We could potentially achieve the _aesthetic_ of this by having a module that imports all of the routers, potentially applies CRUD routes to them, and then that is what `app.ts` imports but that seems kind of convoluted just to save a few lines of import statements to me. Take a look if you like

[entity_routes.md](https://github.com/fbl773/point-b_ank/files/15193479/entity_routes.md)

Reply:
non issue, I'd leave it for now.